api/client/nic.cc

```diff
@@ -271,6 +271,37 @@ static int sub_nic_sg_add(int argc, char **argv,
     return sg_update(nic, all_sg, options);
 }
 
+static void sub_nic_sg_set_help(void) {
+    cout << "usage: butterfly nic sg set NIC SG [SG...] [options...]" << endl;
+    cout << endl << "Set security groups from a vnic" << endl;
```

Comment:
Also, would rewrite commit title to `client: add new sub-command 'nic sg set'` because `set` is ambiguous in your commit title